pkg/scheduler/api/node_info.go

```diff
@@ -90,6 +90,7 @@ func (ni *NodeInfo) SetNode(node *v1.Node) {
 	ni.Allocatable = NewResource(node.Status.Allocatable)
 	ni.Capability = NewResource(node.Status.Capacity)
 	ni.Idle = NewResource(node.Status.Allocatable)
+	ni.Used = EmptyResource()
```

Comment:
If `ni.Node == nil`, we will not update `ni.Used` in `AddTask`; so it seems not necessary to reset `ni.Used`.

Reply:
we need to handle node update case.

Reply:
Yes, that is what I mean